buildSrc/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +0,0 @@
-distributionBase=GRADLE_USER_HOME
```

Comment:
Are the toplevel `gradle.properties` also re-used for the `buildSrc` build now? This would allow us to remove this workaround, too: https://github.com/gradle/gradle/blob/29dcd9a47981b4ffc1b21353063be3c4fdcb38ef/buildSrc/build.gradle.kts#L120-L134

Reply:
From my tests, importing the build and running Gradle tasks from IntelliJ spawns a single daemon.

One thing that might get you into trouble is importing the `buildSrc` build itself, which makes sense and shouldn't be done.

But, the fact that the exception message mention CI is confusing to me. Shouldn't happen on CI, right?

Reply:
Ahh, I forgot about the CI use-case again...

Do I understand you correctly that for IDEA we don't need the workaround any more - importing the project into IDEA does not cause to Gradle daemons to be spawned even without `buildSrc/gradle.properties` and `buildSrc/gradle/wrapper/gradle-wrapper.properties`?

We run `cd buildSrc; ../gradlew clean` on CI to make sure nothing strange is happening in buildSrc (we had some caching problems). This resulted in spawning a separate daemon.
Until we remove cleaning `buildSrc` again we need to keep the workaround for `buildSrc/gradle.properties` in place I guess.

Reply:
> Do I understand you correctly that for IDEA we don't need the workaround any more - importing the project into IDEA does not cause to Gradle daemons to be spawned even without buildSrc/gradle.properties and buildSrc/gradle/wrapper/gradle-wrapper.properties?

Actually no, my bad. Two daemons spawned by IntelliJ if no `buildSrc/gradle.properties` file.
So, the solved issue is only about the Gradle version.

> We run cd buildSrc; ../gradlew clean on CI to make sure nothing strange is happening in buildSrc (we had some caching problems). This resulted in spawning a separate daemon.
> Until we remove cleaning buildSrc again we need to keep the workaround for buildSrc/gradle.properties in place I guess.

Given the above, yes we need to keep this `buildSrc/gradle.properties` file, for both CI and IntelliJ use cases.